packages/hooks/src/useIsHubRedirectionEnabled.ts

```diff
@@ -13,14 +15,16 @@ const useIsHubRedirectionEnabled = () => {
     const { client } = useStore();
     const { account_settings, clients_country } = client;
 
+    const cookieCountry = JSON.parse(Cookies.get('client_information') || '{}')?.residence;
+
+    const userCountry = cookieCountry || account_settings.country_code;
+
     const isHubRedirectionEnabled =
         typeof hubEnabledCountryList === 'object' &&
         hubEnabledCountryList !== null &&
         Array.isArray((hubEnabledCountryList as THubEnabledCountryList).hub_enabled_country_list) &&
-        account_settings.country_code &&
-        (hubEnabledCountryList as THubEnabledCountryList).hub_enabled_country_list.includes(
-            account_settings.country_code
-        );
+        userCountry &&
```

Comment:
nice solution, how did we figure out that it was country_code from client store was the issue

Reply:
@vinu-deriv  Initially, we suspected that the issue was due to a delayed response from GrowthBook. However, upon investigation, we found that the data from GrowthBook is actually available in time. Despite this, the incorrect behavior persists. This led us to look into other factors influencing the logic, and we discovered that account_settings.country_code is taking a long time to populate.

cc: @nijil-deriv